packages/babel-cli/src/babel/options.js

```diff
@@ -4,7 +4,7 @@ import fs from "fs";
 
 import commander from "commander";
 import { version } from "@babel/core";
-import uniq from "lodash/uniq";
+import uniq from "lodash.uniq";
```

Comment:
Is there any chance we can use `Array.from(new Set([]))` and get rid of this dep?

Reply:
That looks feasible, yep - thanks!